### PR TITLE
chore!: set required node version

### PR DESCRIPTION
**Pull request by Caele**
## Requirements checklist

- [x] Api specification
  - [x] Ran `yarn spec`
    - [ ] No changes
     ***OR***
    - [x] API changes has been formally approved
- [x] Unit/Component test coverage
- [x] Correct PR title for the changes (fix, chore, feat)

When build and tests have passed:
- [ ] Add code reviewers, for example @qlik-oss/nebula-core

### CODE REVIEW

**Comment by Caele** [comment]
Makes the next versions a breaking one.

**Review by veinfors — APPROVED** [review]

File: commands/create/templates/sn/none/_package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "keywords": ["qlik", "nebula", "stardust"],
   "files": ["dist"],
   "engines": {
-    "node": ">=8"
+    "node": ">=18"
```
Comment (LiKang6688) [inline commands/create/templates/sn/none/_package.json:13]: why does node 17 not work?

Comment (LiKang6688) [reply commands/create/templates/sn/none/_package.json:13]: I see. Thx!

**Review by LiKang6688 — APPROVED** [review]

File: commands/create/templates/sn/none/_package.json (hunk 1 above)
Comment (LiKang6688) [reply commands/create/templates/sn/none/_package.json:13]: await fetch(). good stuff.

**Pull request merged**